spec/namespaces/security_analytics.yaml

```diff
@@ -0,0 +1,234 @@
+      schema:
+        type: string
+    security_analytics.get_findings::query.detectorType:
+      name: detectorType
+      in: query
+      description: The type of detector used to fetch alerts. Optional when the `detector_id` is specified. Otherwise required.
+      schema:
+        type: string
+    security_analytics.get_findings::query.sortOrder:
+      name: sortOrder
+      in: query
+      description: The order used to sort the list of findings. Possible values are asc or desc. Optional.
+      schema:
+        $ref: '../schemas/_common.yaml#/components/schemas/SortOrder'
+      required: false
+    security_analytics.get_findings::query.size:
+      name: size
+      in: query
+      description: The maximum number of results returned in the response. Optional.
+      schema:
+        type: integer
+        format: int64
+      required: false
+    security_analytics.get_findings::query.startIndex:
+      name: startIndex
+      in: query
+      description: The pagination index. Optional.
+      schema:
+        type: integer
+        format: int64
+      required: false
+    security_analytics.get_findings::query.detectionType:
+      name: detectionType
+      in: query
+      description: The detection rule type that dictates the retrieval type for the findings. When the detection type is threat, it fetches threat intelligence feeds. When the detection type is rule, findings are fetched based on the detector’s rule. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.get_findings::query.severity:
+      name: severity
+      in: query
+      description: The severity for which to retrieve findings. Severity can be critical, high, medium, or low. Optional.
+      schema:
+        $ref: '../schemas/security_analytics.findings.yaml#/components/schemas/Severity'
+      required: false
+    security_analytics.search_finding_correlations::query.finding:
+      name: finding
+      in: query
+      description: The finding ID for which you want to find other findings that are correlated. Required.
+      schema:
+        type: string
+      required: true
+    security_analytics.search_finding_correlations::query.detector_type:
+      name: detector_type
+      in: query
+      description: The log type of findings you want to correlate with the specified finding. Required.
+      schema:
+        type: string
+      required: true
+    security_analytics.search_finding_correlations::query.nearby_findings:
+      name: nearby_findings
+      in: query
+      description: The number of nearby findings you want to return. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.search_finding_correlations::query.time_window:
+      name: time_window
+      in: query
+      description: Sets a time window in which all of the correlations must have occurred together. Optional.
+      schema:
+        type: string
```

Comment:
This can potentially be `$ref: '../schemas/_common.yaml#/components/schemas/Duration`